src/org/spdx/licenselistpublisher/licensegenerator/LicenseTextFormatWriter.java

```diff
@@ -63,21 +63,28 @@ public File getTextFolder() {
 	public void setTextFolder(File textFolder) {
 		this.textFolder = textFolder;
 	}
-
-	/* (non-Javadoc)
-	 * @see org.spdx.licenselistpublisher.licensegenerator.ILicenseFormatWriter#writeLicense(org.spdx.rdfparser.license.SpdxListedLicense, boolean)
+	
+	/**
+	 * @param license license to write
+	 * @param deprecated true if deprecated
+	 * @param deprecatedVersion version deprecated in
+	 * @param wordWrapText if true, reformat the license text wrapping words
+	 * @throws InvalidSPDXAnalysisException 
+	 * @throws IOException 
 	 */
-	@Override
-	public void writeLicense(SpdxListedLicense license, boolean deprecated, String deprecatedVersion) throws IOException, InvalidSPDXAnalysisException {
+	public void writeLicense(SpdxListedLicense license, boolean deprecated, String deprecatedVersion, boolean wordWrapText) throws InvalidSPDXAnalysisException, IOException {
 		String licBaseHtmlFileName = LicenseHtmlFormatWriter.formLicenseHTMLFileName(license.getLicenseId());
 		if (deprecated) {
 			licBaseHtmlFileName = "deprecated_" + licBaseHtmlFileName;
 		}
 		Path textFilePath = Paths.get(textFolder.getPath(), licBaseHtmlFileName + ".txt");
+		if (wordWrapText) {
+			
+		}
```

Comment:
But if you merge this now, we should probably drop this dead code.

Reply:
Got it (I misunderstood the previous comment).

I added that parameter to make it easy to change our mind.  I'm worried others may raise concerns that the plain text is no longer word-wrapped.

I'm thinking we leave this in for one release of the License List, then remove it.  Having the (hopefully) dead code is a bit distracting and I would like to remove it eventually.

Reply:
> Got it (I misunderstood the previous comment).

Ah, no, my original comment wasn't referring to the dead `if`-statement, but to the code that performs (conditional) word-wrapping as a whole. If we never make use of it from now on, we could remove the word-wrapping feature altogether.

> I'm worried others may raise concerns that the plain text is no longer word-wrapped.

It would seem odd to me if people complained about (eventually) missing word wrapping when they were fine with the otherwise broken formatting (compared to upstream) before.

> I'm thinking we leave this in for one release of the License List, then remove it.

Just in my personal option, you should *never* commit dead code. One of the issues it has is that it's totally unclear what its intention is, and whether maybe some meaningful code (here, inside the `if`- statement) was accidentally forgotten and missed during review.

Reply:
> Ah, no, my original comment wasn't referring to the dead if-statement, but to the code that performs (conditional) word-wrapping as a whole. 

OK - seeing the actual code helped - I misunderstood again.

This time, I think I got it.  Completely agree the code should be removed.  I just missed removing that previously.

Just added a commit to remove it.